test/helpers/kubectl.go

```diff
@@ -66,8 +74,35 @@ func CreateKubectl(vmName string, log *logrus.Entry) *Kubectl {
 		return nil
 	}
 	node.logger = log
+
+	f := &rest.Config{}
+	err := rest.SetKubernetesDefaults(f)
+	if err != nil {
+		ginkgo.Fail(fmt.Sprintf(
+			"Cannot set kubernetes configuration defaults: %s", err), 1)
+		return nil
```

Comment:
Not blocking, but IMO this further warrants us to use `--failFast` in our Ginkgo tests,  as returning `nil` and using `ginkgo.Fail` results in the `Kubectl` returned by this function to be `nil`, meaning that if the `BeforeEach` for a test with multiple `It`s fails during this function, the `Kubectl` used in subsequent `It`s under the same `Describe` will be `nil`, and the test framework will panic.